typescript/infra/scripts/verify-validator.ts

```diff
@@ -1,4 +1,5 @@
 import { GetObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import * as Process from 'process';
 import yargs from 'yargs';
 
 const MAX_MISSING_CHECKPOINTS = 10;
```

Comment:
nit: consider `MAX_CONSECUTIVE_MISSING_CHECKPOINTS`

Reply:
I will update to `MAX_CHECKPOINTS_MISSING_IN_A_ROW` to be consistent with the above. Kinda verbose but I guess it is more clear.